### ISSUE TITLE
"Add Space" dialog: initial focus is not visible

### ISSUE THREAD

**Issue by erikjv** [owncloud/client#11870]
When opening the "Add Space" wizard, there is no indication which widget has the focus. It is actually the spaces list, because hitting arrow-down will select the first space. But before doing this, there is no focus/selection indicator. Hitting tab will show the focus going to the buttons, which might lead to confusion on what the first focused item is.

Maybe we should always select the first item in the list when we show the dialog, indicating where the focus is.
